package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "circles-api",
-  "version": "1.7.0-pre-47226e2",
+  "version": "v1.6.0-54-g18aadd5",
```

Comment:
we write version number here without the "v".

Reply:
yeah that's changed with the new commit

Reply:
great! let's create a new testing version so that I can use it for running the core tests

Reply:
this is already available on docker